static/app/views/issueDetails/streamline/activitySection.tsx

```diff
@@ -1,26 +1,92 @@
-import {Fragment} from 'react';
+import {Fragment, useCallback, useMemo, useState} from 'react';
 import styled from '@emotion/styled';
 
+import {addErrorMessage, addSuccessMessage} from 'sentry/actionCreators/indicator';
+import {NoteInputWithStorage} from 'sentry/components/activity/note/inputWithStorage';
+import type {
+  TContext,
+  TData,
+  TError,
+  TVariables,
+} from 'sentry/components/feedback/useMutateActivity';
+import useMutateActivity from 'sentry/components/feedback/useMutateActivity';
 import Timeline from 'sentry/components/timeline';
 import TimeSince from 'sentry/components/timeSince';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import GroupStore from 'sentry/stores/groupStore';
+import type {NoteType} from 'sentry/types/alerts';
 import type {Group} from 'sentry/types/group';
+import type {User} from 'sentry/types/user';
+import {uniqueId} from 'sentry/utils/guid';
+import type {MutateOptions} from 'sentry/utils/queryClient';
 import useOrganization from 'sentry/utils/useOrganization';
+import {useTeamsById} from 'sentry/utils/useTeamsById';
 import {groupActivityTypeIconMapping} from 'sentry/views/issueDetails/streamline/groupActivityIcons';
 import getGroupActivityItem from 'sentry/views/issueDetails/streamline/groupActivityItem';
 
 function StreamlinedActivitySection({group}: {group: Group}) {
   const organization = useOrganization();
+  const teams = useTeamsById().teams;
+
+  const [inputId, setInputId] = useState(uniqueId());
+
+  const me = ConfigStore.get('user');
+  const projectSlugs = group?.project ? [group.project.slug] : [];
+  const noteProps = {
+    minHeight: 140,
+    group,
+    projectSlugs,
+    placeholder: 'Add a comment...',
+  };
+
+  const mutators = useMutateActivity({
+    organization,
+    group,
+  });
+
+  const createOptions: MutateOptions<TData, TError, TVariables, TContext> =
```

Comment:
The types export will be a bit simpler if you export something like

```ts
export type MutateActivityOptions = MutateOptions<TData, TError, TVariables, TContext>
```

in the other file and just use that here, since we don't care about those `T` types anyway.